config/config.txt

```diff
@@ -98,7 +99,7 @@ ALLOW_RANDOM_EVENTS
 TRAITOR_SCALING
 
 ## if objectives are disabled
-#OBJECTIVES_DISABLED
+OBJECTIVES_DISABLED

```

Comment:
I think it should be changed, objectives can be changed, and gives people a starting point for roleplay, they can ahelp if they don't want to do it etc